src/SprykerEco/Zed/Econda/Business/Collector/File/EcondaProductCollector.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Collector\File;
+
+use Generated\Shared\Transfer\LocaleTransfer;
+use Generated\Shared\Transfer\StorageProductImageTransfer;
+use Orm\Zed\Category\Persistence\Map\SpyCategoryTableMap;
+use Propel\Runtime\ActiveQuery\Criteria;
+use Propel\Runtime\Collection\Collection;
+use Spryker\Shared\SqlCriteriaBuilder\CriteriaBuilder\CriteriaBuilderInterface;
+use Spryker\Zed\ProductCategory\Persistence\ProductCategoryQueryContainerInterface;
+use Spryker\Zed\ProductImage\Persistence\ProductImageQueryContainerInterface;
+use SprykerEco\Zed\Econda\Business\Collector\AbstractDatabaseCollector;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPriceBridgeInterface;
+use SprykerEco\Zed\Econda\EcondaConfig;
+use SprykerEco\Zed\Econda\Persistence\Econda\AbstractPdoEcondaQuery;
+
+class EcondaProductCollector extends AbstractDatabaseCollector
+{
+    //CSV file columns
+    const ID_COLUMN = 'ID';
+    const NAME_COLUMN = 'Name';
+    const DESCRIPTION_COLUMN = 'Description';
+    const PRODUCTURL_COLUMN = 'ProductURL';
+    const IMAGE_URL_COLUMN = 'ImageURL';
+    const PRICE_COLUMN = 'Price';
+    const STOCK_COLUMN = 'Stock';
+    const PRODUCT_CATEGORY_COLUMN = 'ProductCategory';
+
+    //internal query fields
```

Comment:
Comments are usually a sentence (and leading space before start of sentence): `// Internal ...`
Compared to commented out code that would indeed look like this here.